services/api/service.go

```diff
@@ -950,15 +1108,36 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		}
 	}
 
-	builderIsHighPrio, builderIsBlacklisted, err := api.redis.GetBlockBuilderStatus(payload.Message.BuilderPubkey.String())
+	builderStatus, err := api.redis.GetBlockBuilderStatus(payload.Message.BuilderPubkey.String())
 	log = log.WithFields(logrus.Fields{
-		"builderIsHighPrio":    builderIsHighPrio,
-		"builderIsBlacklisted": builderIsBlacklisted,
+		"builderStatus": builderStatus.String(),
 	})
 	if err != nil {
 		log.WithError(err).Error("could not get block builder status")
 	}
 
+	// Check for collateral in optimistic case.
+	if builderStatus == common.OptimisticActive {
+		builderCollateralStr, err := api.redis.GetBlockBuilderCollateral(payload.Message.BuilderPubkey.String())
+		if err != nil {
+			log.WithError(err).Error("could not get block builder collateral string")
+			builderCollateralStr = ""
```

Comment:
Should this be `builderCollateralStr = "0"`, at least for clarity that by default the collateral value is 0?

Reply:
ya you are right :-) https://go.dev/play/p/wEJKRYaiB95